config.yaml

```diff
@@ -71,16 +73,27 @@ services:
       # is also configured.
       stream_field: meta.stream
 
+      # If set, events will be written to this file.
+      # If set to 'stdout', events will be written to stdout.
+      output_path: stdout
+
       # kafka configs go here.
-      kafka:
-        conf:
-          metadata.broker.list: 'localhost:9092'
-        # kafka topic conf goes here
-        topic_conf: {}
+      # If kafka.conf is defined, make sure the optionalDependency
+      # node-rdkafka-factory (and node-rdkafka) is installed properly.
+      # kafka:
+      #  conf:
+      #    metadata.broker.list: 'localhost:9092'
+      #    # If this is set, node-rdkafka-statsd will be used
```

Comment:
maybe no need to add a comment within a comment ( e.g. multitude of # symbols)

Reply:
I reformatted a bit and left a couple in.  Its nice to be able to uncomment the block and still have a few comments in there.